### DISCUSSION TITLE
gen_coords doesn't read the input of -b

### DISCUSSION THREAD

**Discussion by NoahG177** [marrink-lab/polyply_1.0#373] [Q&A] [answered]
Our protein has two N-glycosylation sites, and we are trying to add two of the same glycan. We have generated all our topologies files as indicated on the polyply tutorials. However the command
 
polyply gen_coords -p mouse_only.top -o protein_glycans_new.gro -name Glycoprotein -c mouse_cg_centered.gro -b build_file.bld -back_fudge 0.8 -dens 1000

does not add the glycans. With or without -b the output .gro is identical to the input .gro. Would anyone have any input on what might be going on?

**Comment by fgrunewald** [comment]
@NoahG177 could you share some of your input files. It is really hard to debug without knowing how the itps look like. Does your protein consist of multiple chains?

**Reply by NoahG177** [reply]
Thank you for your response. Our protein only has one chain. Here is what we are doing:

polyply gen_coords -p mouse_only.top -o protein_glycans_new.gro -name Glycoprotein -c mouse_cg_centered.gro -b build_file.bld -back_fudge 0.8 -dens 1000

[myfile.tar.gz](https://github.com/user-attachments/files/15791368/myfile.tar.gz)

Here is the tar compressed file including our input files.

**Reply by NoahG177** [reply, marked as answer]
Nevermind we figured it out! in the .top file one should add the keyword COMBINED , which for us corresponds to our protein+glycans, under [molecules] instead of molecule_0 which for us corresponds to the protein alone.

**Reply by fgrunewald** [reply]
If the issues resolve themselves that's the best kind of issue!  Glad you found the mistake.
